uicomponent-compose/timetable2021/src/main/java/io/github/droidkaigi/feeder/timetable2021/TimetableViewModel.kt

```diff
@@ -25,7 +25,7 @@ interface TimetableViewModel :
 
     sealed class Event {
         class ChangeFavoriteFilter(val filters: Filters) : Event()
-        class ToggleFavorite(val session: TimetableItem.Session) : Event()
+        class ToggleFavorite(val timetableItem: TimetableItem) : Event()
```

Comment:
@takahirom 
[proposal]
Can we use the favorite feature for `TimetableItem.Special` ?

Reply:
👀

Reply:
According to design, yes 👍 
![image](https://user-images.githubusercontent.com/1386930/134452023-dcd3bcaa-a81e-4441-94dc-1c099221fbe6.png)